### PR TITLE
Game dev config

### PR DESCRIPTION
**Pull request by AtlantaDancer**
Here is a cleaner solution for configuration settings.  I am using config.json file - the script copies either the DEV version (pointed to local host) or prod version which points to a live IP address (currently the one that in the file before http://152.46.19.3:443).  It is loaded in the load screen and utilized in the game scene.

### CODE REVIEW

**Comment by AtlantaDancer** [comment]
I took a look and found the gulp task `gulp-json-modify`.  This will allow us to have a single config file and for the dev/test build replace the host/port information with localhost:3030.  I will update the request with this update and the change to the loader scene.  In the wiki or instructions we can mention to test local update the gulp file.  I can add variables at the top of the gulpfile.js for the local server to use for testing with comments to make it more obvious what to change if needed.

File: game/gulpfile.js [hunk 1]
```diff
@@ -11,6 +11,19 @@ let babelify = require('babelify');
 let source = require('vinyl-source-stream');
 let buffer = require('vinyl-buffer');
 let path = require('path');
+let rename = require('gulp-rename');
```
Comment (ignacioxd) [inline game/gulpfile.js]: Looks like this dependency is no longer needed

File: config/config.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "protocol": "http",
+  "host": "152.46.18.48",
```
Comment (ignacioxd) [inline config/config.json]: Maybe this should be renamed to `raginmages.net` to keep it consistent with the instructions below.

File: game/gulpfile.js [hunk 3]
```diff
@@ -11,6 +11,19 @@ let babelify = require('babelify');
...
+let jsonModify = require('gulp-json-modify');
+
+/******* NOTE: When testing using SocketIO to your local server set the variables below to the SocketIO server
+        That you wish to communicate with
+        This setting will be used when you do "yarn start" for socket communication
+        If you do a yarn build it will use the production server raginmages.com
```
Comment (ignacioxd) [inline game/gulpfile.js]: raginmages.net :)

**Comment by ignacioxd** [comment]
Looks like there is still a conflict in one of the files. Could you please take a look?

**Pull request merged**